bin/migrate

```diff
@@ -1,115 +1,110 @@
 #!/usr/bin/env node
+'use strict'
 
 /**
  * Module dependencies.
  */
 
 var migrate = require('../')
-  , join = require('path').join
-  , fs = require('fs')
-  , dateFormat = require('dateformat');
+var chalk = require('chalk')
+var join = require('path').join
+var fs = require('fs')
+var dateFormat = require('dateformat')
 
 /**
  * Arguments.
  */
 
-var args = process.argv.slice(2);
+var args = process.argv.slice(2)
 
 /**
  * Option defaults.
  */
 
-var options = { args: [] };
-
-/**
- * Current working directory.
- */
-
-var cwd;
+var options = { args: [] }
 
 /**
  * Usage information.
  */
 
 var usage = [
-    ''
-  , '  Usage: migrate [options] [command]'
-  , ''
-  , '  Options:'
-  , ''
-  , '     -c, --chdir <path>      change the working directory'
-  , '     --state-file <path>     set path to state file (migrations/.migrate)'
-  , '     --template-file <path>  set path to template file to use for new migrations'
-  , '     --date-format <format>  set a date format to use for new migration filenames'
-  , '  Commands:'
-  , ''
-  , '     down   [name]    migrate down till given migration'
-  , '     up     [name]    migrate up till given migration (the default command)'
-  , '     create [title]   create a new migration file with optional [title]'
-  , ''
-].join('\n');
+  '',
+  '  Usage: migrate [options] [command]',
+  '',
+  '  Options:',
+  '',
+  '     -c, --chdir <path>      change the working directory',
+  '     --state-file <path>     set path to state file (migrations/.migrate)',
+  '     --template-file <path>  set path to template file to use for new migrations',
+  '     --date-format <format>  set a date format to use for new migration filenames',
+  '  Commands:',
+  '',
+  '     down   [name]    migrate down till given migration',
+  '     up     [name]    migrate up till given migration (the default command)',
+  '     create [title]   create a new migration file with optional [title]',
+  ''
+].join('\n')
 
 /**
  * Migration template.
  */
 
 var template = [
-    '\'use strict\''
-  , ''
-  , 'exports.up = function(next) {'
-  , '  next();'
-  , '};'
-  , ''
-  , 'exports.down = function(next) {'
-  , '  next();'
-  , '};'
-  , ''
-].join('\n');
+  '\'use strict\'',
+  '',
+  'exports.up = function(next) {',
+  '  next();',
+  '};',
+  '',
+  'exports.down = function(next) {',
+  '  next();',
+  '};',
```

Comment:
I personally think we should drop the semicolons in the template as well, this will be a major version anyhow :)